src/api/currency/content-types/currency/schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "kind": "collectionType",
+  "collectionName": "currencies",
+  "info": {
+    "singularName": "currency",
+    "pluralName": "currencies",
+    "displayName": "Financial.CurrencyConversion"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "year": {
+      "type": "string",
+      "default": "2020",
```

Comment:
**question:** Do we need to have a default value here? If so, is there a reason to pick 2020 specifically?

Reply:
Removed the default value for now.